nebuchadnezzar/nebu/h5p_injection.py

```diff
@@ -253,41 +248,39 @@ def add_tag(name, value):
     return tags
 
 
-def load_h5p_interactive(interactive_path: str, private_path: str):
+def load_h5p_interactive(
+    interactive_path: str, private_path: str | None = None
+):
     metadata_path = os.path.join(interactive_path, "metadata.json")
     content_path = os.path.join(interactive_path, "content.json")
     h5p_path = os.path.join(interactive_path, "h5p.json")
-    private_metadata_path = os.path.join(private_path, "metadata.json")
-    private_content_path = os.path.join(private_path, "content.json")
     if (
         not os.path.exists(metadata_path) or
         not os.path.exists(content_path) or
         not os.path.exists(h5p_path)
     ):
         logger.error(f"MISSING INTERACTIVE DATA: {interactive_path}")
         return None
-    h5p_in = {}
-    h5p_in["h5p"] = json.loads(Path(h5p_path).read_bytes())
-    h5p_in["content"] = recursive_merge(
-        json.loads(Path(content_path).read_bytes()),
-        json.loads(Path(private_content_path).read_bytes())
-        if os.path.exists(private_content_path)
-        else {},
-    )
-    h5p_in["metadata"] = recursive_merge(
-        json.loads(Path(metadata_path).read_bytes()),
-        json.loads(Path(private_metadata_path).read_bytes())
-        if os.path.exists(private_metadata_path)
-        else {},
-    )
-    return h5p_in
+    h5p = json.loads(Path(h5p_path).read_bytes())
+    content = json.loads(Path(content_path).read_bytes())
+    metadata = json.loads(Path(metadata_path).read_bytes())
+    if private_path is not None:
```

Comment:
When would private_path be used for exercise injection? Maybe this is left in for future use-cases (like building a teacher answer-key or something?)